src/content/docs/reference/imports.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Import Directives
+---
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import ParameterLayout from '../../../components/parameter_layout.astro';
+import {formats} from '../../lib/formats.ts';
+import FormatTable from '../../../components/format_table.astro';
+
+--- 
+
+Import directives are used to import external data resources as predicates.
+
+## Syntax
+The general syntax of an import statement is as follows:
+```
+@import <predicate> :- <format>{<parameter>=<value>, ...} .
+```
+In the example below `table` denotes the predicate that data is imported to, `csv` is the data format (comma-separated values),
+and `path/to/file.csv` is the relative path to the file. Instead of a local path, it is also possible to define a URL from
+which data should be downloaded, e.g., `resource = "https://example.org/file.csv"`.
+
+```
+@import table :- csv { resource="path/to/file.csv", format=(int,string, any)}.
+```
+
+
+## Formats
+
+Currently the following formats are supported:
+
+<table class="w-full table-auto border-separate">
+  <thead>
+    <tr>
+      <th class="pe-8 text-left w-32 whitespace-nowrap">Format</th>
+      <th class="text-left">Parameters</th>
+    </tr>
+  </thead>
+  <tbody>
+    {Object.entries(formats).map(([key, format]) => (
+        <FormatTable of={key} direction='import' />
+      ))}
+  </tbody>
+</table>
+
```

Comment:
this shares a lot of code with the export version, with the only difference here being the `'import'` vs `'export'`. Can we pull this out into another component that takes a `Direction`?